docs/_layouts/variation.html

```diff
@@ -32,13 +32,15 @@ <h3 class="h5 o-expandable_header-left o-expandable_label">
     </aside>
         {% include variation-content.html %}
         <div class="o-editor_link" id="edit-page">
-            <a class="a-btn" href="{{ site.baseurl }}/admin/#/collections/{{ page.section }}/entries/{{ page.path | split: '/' | last | replace: '.md', '' }}" title="Edit this page in Netlify CMS">
+            <a class="a-btn" href="{{ "/admin/#/collections/" | append: page.section | append: "/entries/" | append: page.path | split: '/' | last | replace: '.md' | relative_url }}" title="Edit this page in Netlify CMS">
```

Comment:
This is returning `/design-system/<page-name>` instead of `/design-system/admin/#/collections/<page-section>/entries/<page-name>`. Try:

```suggestion
            <a class="a-btn" href="{{ '/admin/#/collections/' | append: page.section | append: '/entries/' | relative_url }}{{ page.path | split: '/' | last | replace: '.md' }}" title="Edit this page in Netlify CMS">
```